### DISCUSSION TITLE
colapse left sidebar in v3 ?

### DISCUSSION THREAD

**Discussion by sandermanneke** [filamentphp/filament#7965] [Help] [answered]
In V2 you could make the sidebar collapsible for desktop, when trying that in V3 that no longer seems to work:
https://filamentphp.com/docs/2.x/admin/appearance#collapsible-sidebar

Can I make the sidebar collapsible for desktop in V3 ?

**Comment by danharrin** [comment, marked as answer]
In the panel provider `sidebarCollapsibleOnDesktop()`
